src/jit/morph.cpp

```diff
@@ -7817,8 +7817,14 @@ void Compiler::fgMorphTailCall(GenTreeCall* call)
         unsigned vtabOffsOfIndirection;
         unsigned vtabOffsAfterIndirection;
         bool     isRelative;
+        bool     isRelativeAfterIndirection;
         info.compCompHnd->getMethodVTableOffset(call->gtCallMethHnd, &vtabOffsOfIndirection, &vtabOffsAfterIndirection,
-                                                &isRelative);
+                                                &isRelative, &isRelativeAfterIndirection);
+
+        if (isRelativeAfterIndirection)
```

Comment:
Two separate flags could be useful for enabling more or less aggressive memory optimizations for FNV. Now there are next possible combinations: 
`isRelative==false` and  `isRelativeAfterIndirection==false`
`isRelative==true` and  `isRelativeAfterIndirection==false`
`isRelative==true` and  `isRelativeAfterIndirection==true`

Reply:
I understand that it is more flexibility, but I doubt that we will ever take advantage of it and it increases the number of options one has to worry about when working on the code.

Reply:
We target different devices and this option might be useful, as memory reduction of this optimization comes with performance reduction on benchmarks. 
Maybe this and similar places could be refactored, so that it will be easier to maintain them. What do you think about this?

Reply:
Historically, fine grained controls like these did not survive for long. I think we just need two options: fully position independent data structures on/off. Similar to how position-independent-code works in C/C++. You either turn it on or off. There are no fine-grained controls.

Reply:
Fixed